challenges/02 It's super effective/app.js

```diff
@@ -1 +1,157 @@
+const typeCharactaristics = {
+    normal: {rock: 0.5, ghost: 0, steel: 0.5},
+    fire: {fire: 0.5, water: 0.5, grass: 2, ice: 2, bug: 2, rock: 0.5, dragon: 0.5, steel: 2},
+    water: {fire: 2, water: 0.5, grass: 0.5, ground: 2, rock: 2, dragon: 0.5},
+    electric: {water: 2, electric: 0.5, ground: 0, fly: 2, dragon: 0.5},
+    grass: {fire: 0.5, water: 2, grass: 0.5, poison: 0.5, ground: 2, fly: 0.5, bug: 0.5, rock: 2, dragon: 0.5, steel: 0.5},
+    ice: {fire: 0.5, water: 0.5, grass: 2, ice: 0.5, ground: 2, fly: 2, dragon: 2, steel: 0.5},
+    fighting: {normal: 2, ice: 2, poison: 0.5, fly: 0.5, psychic: 0.5, bug: 0.5, rock: 2, ghost: 0, dragon: 2, steel: 2, fairy: 0.5},
+    poison: {grass: 2, poison: 0.5, ground: 0.5, rock: 0.5, ghost: 0.5, steel: 0, fairy: 2},
+    ground: {fire: 2, electric: 2, grass: 0.5, poison: 2, flying: 0, bug: 0.5, rock: 2, steel: 2},
+    flying: {electric: 0.5, grass: 2, fighting: 2, bug: 2, rock: 0.5, steel: 0.5},
+    psychic: {fighting: 2, poison: 2, psychic: 0.5, dark: 0, steel: 0.5},
+    bug: {fire: 0.5, grass: 2, fighting: 0.5, poison: 0.5, fly: 0.5, psychic: 2, ghost: 0.5, dark: 2, steel: 0.5, fairy: 0.5},
+    rock: {fire: 2, ice: 2, fighting: 0.5, ground: 0.5, fly: 2, bug: 2, steel: 0.5},
+    ghost: {normal: 0, psychic: 2, ghost: 2, dark: 0.5},
+    dragon: {dragon: 2, steel: 0.5, fairy: 0},
+    dark: {fighting: 0.5, psychic: 2, ghost: 2, dark: 0.5, fairy: 0.5},
+    steel: {fire: 0.5, water: 0.5, electric: 0.5, ice: 2, rock: 2, steel: 0.5, fairy: 2},
+    fairy: {fire: 0.5, fighting: 0.5, poison: 0.5, ground: 2, dragon: 2, steel: 0.5}
+}
+
+/**Speichert alle Requests ab, die später nötig sind. */
+var requestsToDo = [undefined, undefined];
+
+/**
+ * Stellt eine Frage im Terminal und erwartet eine Antwort.
+ * Auf Basis der Antwort wird der Vergleich der Typen vorgenommen und
+ * wenn nötig, ein Api-Aufruf um Attacken-und Pokemon-Objekte abzurufen.
+ */
+rl.question('Lass den Vergleich beginnen: ', (answer) => {
+    let actors = splitAnswer(answer);
+    checkComposition(actors);
+    axios.all(requestsToDo)
+        .then((results) => {
+            if(results[0] != undefined) {
+                actors[0] = results[0].data.type.name;
+            }
+            if(results[1] != undefined) {
+                let pokemonTypes = [];
+                results[1].data.types.forEach(type => {
+                    pokemonTypes.push(type.type.name)
+                })
+                actors[1] = pokemonTypes
+            }
+            
+            let effectivityValue = effectivityCalculation(actors);
+            console.log('Der Effekt ist: ' + giveAnswerFromEffectivityValue(effectivityValue));
+            rl.close();
+        })
+})
+
+/**
+ * Unterteilt den eingegebenen String in Angreifer und Verteidiger.
+ *
+ * @param answer {string} - Benutzereingabe mit Angreifer und Verteidiger.
+ * @returns {Array|*} - Enthält auf: Index 0 -> Angreifer.

```

Comment:
Sternchen bedeutet, es kann alles sein, was in Javascript denkbar ist. 
Ich vermute du wolltest damit kennzeichne, dass _im_ Array sich verschiedene Typen (String/Array) befinden, für andere sieht es aber so aus, als würde die Funktion selbst mehr als nur Arrays zurückgeben können. 
Ein Regel hierfür weiß ich auch nicht, persönlich würde ich die extra Information in die Beschreibung zu 'Index 1' packen.